api/base/filters.py

```diff
@@ -377,6 +377,13 @@ def get_filtered_queryset(self, field_name, params, default_queryset):
                 item for item in default_queryset
                 if params['value'].lower() in getattr(item, field_name, {}).lower()
             ]
+        elif isinstance(field, JSONAPIListField):
```

Comment:
@brianjgeiger do you think this could just be `ser.ListField`? Doesn't look like there's any practical difference between the two classes here